4-storage/redshift-storage/jsonpaths/com.amazon.aws.cloudfront/wd_access_log_1.json

```diff
@@ -1,34 +1,35 @@
 {
-  "jsonpaths": [
-
-    "$.schema.vendor",
-    "$.schema.name",
-    "$.schema.format",
-    "$.schema.version",
-
-    "$.hierarchy.rootId",
-    "$.hierarchy.rootTstamp",
-    "$.hierarchy.refRoot",
-    "$.hierarchy.refTree",
-    "$.hierarchy.refParent",
-
-    "$.data.dateTime",
-    "$.data.xEdgeLocation",
-    "$.data.scBytes",
-    "$.data.cIp",
-    "$.data.csMethod",
-    "$.data.csHost",
-    "$.data.csUriStem",
-    "$.data.scStatus",
-    "$.data.csReferer",
-    "$.data.csUserAgent",
-    "$.data.csUriQuery",
-    "$.data.csCookie",
-    "$.data.xEdgeResultType",
-    "$.data.xEdgeRequestId",
-    "$.data.xHostHeader",
-    "$.data.csProtocol",
-    "$.data.csBytes",
-    "$.data.timeTaken"
+    "jsonpaths": [
+        "$.schema.vendor",
+        "$.schema.name",
+        "$.schema.format",
+        "$.schema.version",
+        "$.hierarchy.rootId",
+        "$.hierarchy.rootTstamp",
+        "$.hierarchy.refRoot",
+        "$.hierarchy.refTree",
+        "$.hierarchy.refParent",
+        "$.data.dateTime",
+        "$.data.cIp",
+        "$.data.csBytes",
+        "$.data.csCookie",
+        "$.data.csHost",
+        "$.data.csMethod",
+        "$.data.csProtocol",
+        "$.data.csReferer",
+        "$.data.csUriQuery",
+        "$.data.csUriStem",
+        "$.data.csUserAgent",
+        "$.data.scBytes",
+        "$.data.scStatus",
+        "$.data.sslCipher",
+        "$.data.sslProtocol",
+        "$.data.timeTaken",
+        "$.data.xEdgeLocation",
+        "$.data.xEdgeRequestId",
+        "$.data.xEdgeResponseResultType",
+        "$.data.xEdgeResultType",
+        "$.data.xForwardedFor",
+        "$.data.xHostHeader"
     ]
```

Comment:
I think for this JSON Paths (and associated table) file we should revert to old order with new fields on the end. Otherwise it's quite a difficult upgrade...